docs/Guides/Ecosystem.md

```diff
@@ -557,3 +557,4 @@ section.
 #### [Community Tools](#community-tools)
 - [`fast-maker`](https://github.com/imjuni/fast-maker) route configuration
   generator by directory structure.
+- [`simple-tjscli`](https://github.com/imjuni/simple-tjscli) typescript interface convert json-schema.
```

Comment:
I think this description can be improved. As it is, it is a fragmented sentence that doesn't give a clear indication of what the library is.

Reply:
How about it?

```
cli tool to json-schema generate from typescript interface
```

Reply:
> cli tool to json-schema generate from typescript interface

"CLI tool to generate JSON Schema from TypeScript interfaces."

Reply:
@jsumners Thank you for your help! I changed sentence.